src/MyERC20.sol

```diff
@@ -21,53 +19,36 @@ contract MyToken is ERC20, ERC20Permit, Pausable, Ownable, ENSMetadata {
             "worldtokenproject.eth",
             _ensRegistry
         )
+        Ownable(msg.sender)
     {}
 
     // Minting function to create new tokens
     // Allows the owner to mint additional tokens, increasing the total supply
-    function mint(
-        address to,
-        uint256 amount
-    ) public onlyOwner withContext("Minting new tokens by the owner") {
+    function mint(address to, uint256 amount) public onlyOwner {
         _mint(to, amount);
     }
 
     // Burning function to reduce the total supply
     // Allows a token holder to burn their tokens, decreasing the total supply
-    function burn(
-        uint256 amount
-    ) public withContext("Burning tokens by the holder") {
```

Comment:
@Madgeniusblink I have removed `withContext` modifier, I think it's not ENSMetadata's responsibility to provide such a method. But keeping that inside `MetadataLib` is a right choice. we can internally use that to log sequence of operation or function calls. Please let me know your opinion here.